ibis/backends/tests/test_map.py

```diff
@@ -235,8 +236,8 @@ def test_map_construct_dict(con, keys, values):
 @pytest.mark.notyet(["postgres"], reason="only support maps of string -> string")
 @pytest.mark.notimpl(
     ["flink"],
-    raises=exc.OperationNotDefinedError,
-    reason="No translation rule for <class 'ibis.expr.operations.arrays.Array'>",
+    raises=Py4JJavaError,
+    reason="Map key type should be non-nullable",
```

Comment:
Instead can you add `ops.Map` to the not supported set of operations?

Reply:
Thanks, but I do not know "how to add to the not supported set of operations". Can you please point me to where I can do it?

Reply:
At the bottom of `registry.py`, you can add to `_invalid_operations`.

Reply:
But that would mark `ops.Map` as invalid for Flink backend, would not it? We implement support for it in `ops.Map: _map`.